src/application/components/_applicationFractionalField.scss

```diff
@@ -0,0 +1,21 @@
+.ApplicationFractionField {
+  fieldset {
+    display: flex;
+    margin-inline-end: 0;
+    margin-inline-start: 0;
```

Comment:
TIL that these exist. Not exactly a bleeding edge feature either (added to FF and Chrome in '19-'20), so it should probably be fine to use.

Reply:
I just checked with inspector that these were the properties causing ugly extra margin on the outer sides of the fields. :laughing: So I cancelled those by setting to 0 and it fixed the issue..